helix-core/src/main/java/org/apache/helix/model/ClusterConfig.java

```diff
@@ -1018,6 +1032,22 @@ public void setAbnormalStateResolverMap(Map<String, String> resolverMap) {
     return idealStateRuleMap;
   }
 
+  public void setSuperClusterRebalanceConfig(RebalanceConfig rebalanceConfig) {
+    _record.setMapField(ClusterConfigProperty.SUPER_CLUSTER_REBALANCE_CONFIG.name(),
+        rebalanceConfig.getConfigsMap());
+  }
+
+  public RebalanceConfig getSuperClusterRebalanceConfig() {
+    Map<String, String> rebalanceConfigMap =
+        _record.getMapField(ClusterConfigProperty.SUPER_CLUSTER_REBALANCE_CONFIG.name());
+    if (rebalanceConfigMap != null) {
+      return new RebalanceConfig(rebalanceConfigMap);
+    } else {
```

Comment:
Unnecessary else block

Reply:
It expresses the code logic better, IMO.

I can find many arguments online about this style, but I don't see much difference in this condition. If the previous return is for the exception or some early termination logic, then I think we should not use "else". But here, I don't see much difference. And my mind follows the current code better : )

Reply:
+1 to @NealSun96 . I would suggest to follow the coding convention.

Reply:
Which code convention are we talking about? I think there is no such rule... But please share a link so I can read it.

Reply:
This line is not necessary with the latest code change.